CMakeLists.txt

```diff
@@ -253,7 +253,11 @@ endif()
 add_subdirectory(${PROJECT_SOURCE_DIR}/tt_metal)
 add_subdirectory(${PROJECT_SOURCE_DIR}/ttnn)
 
-add_subdirectory(${PROJECT_SOURCE_DIR}/tests EXCLUDE_FROM_ALL)
+option(TT_METAL_BUILD_TESTS "Enables build of tt_metal tests" OFF)
+option(TTNN_BUILD_TESTS "Enables build of ttnn tests" OFF)
```

Comment:
Maybe put an extra word in here that it's "gtests" for TTNN, as we have both Python and CPP tests. You would think that that would be obvious to people, but you'd be surprised